File: JS/02.Basics/02.3-ConditionalStatementsExercise/07.Shopping.js
function shoppingBudget(budget, videoCardsCount, processorsCount, ramCount) {
    let totalVCPrice = videoCardsCount * 250
    let totalPrPrice = processorsCount * (totalVCPrice * 0.35)
    let totalRAMPrice = ramCount * (totalVCPrice * 0.1)

    let total = totalPrPrice + totalRAMPrice + totalVCPrice

    if (videoCardsCount > processorsCount) {
        total = total * 0.85
    }

    if (budget >= total) {
        let rest = budget - total
        rest = rest.toFixed(2)
        console.log(`You have ${rest} leva left!`)
    }
    else {
        let rest = total - budget
        rest = rest.toFixed(2)
        console.log(`Not enough money! You need ${rest} leva more!`)
    }
}

shoppingBudget("900", "2", "1", "3"); // You have 198.75 leva left!
shoppingBudget("920.45", "3", "1", "1"); // Not enough money! You need 3.92 leva more!

File: C++/01-Basics/02.3-ConditionalStatementsExercise/07.Shopping/07.Shopping.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double budget, videoCards, processors, ram;
    cin >> budget >> videoCards >> processors >> ram;

    double videoCardsCost = videoCards * 250.0;
    double processorsCost = processors * (0.35 * videoCardsCost);
    double ramCost = ram * (0.10 * videoCardsCost);

    double totalCost = videoCardsCost + processorsCost + ramCost;

    if (videoCards > processors)
        totalCost *= 0.85;

    cout << fixed << setprecision(2);

    if (budget >= totalCost)
        cout << "You have " << budget - totalCost << " leva left!" << endl;
    else
        cout << "Not enough money! You need " << totalCost - budget << " leva more!" << endl;

    return 0;
}
